teres/bkr_handlers.py

```diff
@@ -422,27 +431,45 @@ def _emit_file(self, record):
         if not record.flags.get(QUIET_FILE, False):
             self._emit_log(teres.ReportRecord(teres.FILE, msg))
 
-    def _thread_emit_file(self, record, offset=0):
+    def _thread_emit_file(self, record):
         """
         Send file record to beaker.
         """
-        url = self._generate_url(record)
 
-        position = record.logfile.tell()
-        record.logfile.seek(offset)
-        payload = record.logfile.read()
-        record.logfile.seek(position)
-
-        logger.debug("ThinBkrHandler: calling _thread_emit_file with: %s",
-                     record.logname)
+        def suggest_rng_from():
```

Comment:
Please, do not use "rng" as it's abbreviation for "Random Number Generator"

Reply:
renamed to `range`